vcd/resource_vcd_vapp_vm_hot_updates_test.go

```diff
@@ -152,6 +156,7 @@ func TestAccVcdVAppHotUpdateVm(t *testing.T) {
 
 					resource.TestCheckResourceAttr("vcd_vapp_vm."+hotVmName1, `guest_properties.guest.hostname`, "test-host2"),
 
+					resource.TestCheckResourceAttr("vcd_vapp_vm."+hotVmName1, `storage_profile`, params["StorageProfile2"].(string)),
```

Comment:
Can we check that VM `ID` is the same after update. Otherwise we are not even sure if VM supports update or it was re-created as part of this step.

Reply:
in 160 line there is function which checks `testAccCheckVcdVmNotRestarted("vcd_vapp_vm."+hotVmName1, hotVappName, hotVmName1),`